services/bitbucket/bitbucket.tester.js

```diff
@@ -152,35 +145,6 @@ t.create('pr (server, not found)')
     message: 'not found',
   })
 
-t.create('pr (auth)')
```

Comment:
What's the thinking behind removing these tests, and would you anticipate removing the same types of tests in other services?

Reply:
Sorry I was reading from the bottom up, I see this validation is now handled above in the new `spec` file 👍